backend/open_webui/utils/orq_client.py

```
@@ -0,0 +1,71 @@
+                "name": model,
+                "object": "model",
+                "created": int(time.time()),
+                "owned_by": "orq",
+                "orq": model,
+                "connection_type": "orq",
+                "tags": [],
+              }
+            ]
+            
+        except Exception as e:
+            log.error(f"Failed to fetch ORQ models: {e}")
```

Ruff (0.12.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)